src/platform/site-wide/sass/consolidated-patches.scss

```diff
@@ -1,3 +1,20 @@
+.usa-banner-header-expanded {
+  @media (max-width: $medium-screen) {
```

Comment:
Jeff was just talking about how we should try to do our media queries in a mobile-first approach, so using queries that follow the `min-width` pattern instead. I'm not 100% sure that should apply to this teamsite work. But given the nature of these teamsite CSS fixes I'm fine with doing whatever gets the job done.

Reply:
Thanks @erikphansen, this override only applies to smaller screens.